.github/workflows/auto-approve.yml

```diff
@@ -2,7 +2,7 @@
 
 name: auto-approve
 on:
-  pull_request:
+  pull_request_target:
     types: [ labeled, unlabeled, opened, synchronize, reopened, ready_for_review, review_requested ]
```

Comment:
This list of trigger types does however look slightly excessive (not sure this is useful, but also don't think it hurts)

Reply:
Yeah I agree - pretty sure its causing duplicate approvals also...but didn't want to get tangled with that now :)